libyul/AsmAnalysis.cpp

```diff
@@ -497,6 +497,16 @@ bool AsmAnalyzer::operator()(ForLoop const& _for)
 	return success;
 }
 
+bool AsmAnalyzer::operator()(Break const& _break)
```

Comment:
Same question here.

Reply:
Same as above, we explicitly need to specify them. but in this case, I moved the impl into the header (as one-liner), as suggested below.